plugins/module_utils/redfish_utils.py

```diff
@@ -2311,11 +2311,17 @@ def set_bios_attributes(self, attributes):
 
         # Construct payload and issue PATCH command
         payload = {"Attributes": attrs_to_patch}
+        additional_data = ''
+
+        if 'PwrButton' in attrs_to_patch:
+            additional_data = '. Reboot required to apply setting'
```

Comment:
I'd prefer making this a vendor-specific workaround check like we have elsewhere. Technically anyone can implement a "PwrButton" attribute. But to clarify one point, everyone requires a reboot after configuring these settings, but Dell is unique in that they require an addition property to be set to fall in line with the recommended Redfish behavior.

```suggestion
        # WORKAROUND
        # Older Dell systems require manually setting the apply time to
        # "OnReset" to spawn a proprietary job to apply the BIOS settings
        vendor = self._get_vendor()['Vendor']
        if vendor == 'Dell':
```

Reply:
Great suggestion. I wasn't aware of this being possible. I'll push an updated commit.

Reply:
I've removed "Older" from the text, as this is still the case with current Dell systems. Tested on PowerEdge R7525